r/introduction-to-tidy-finance.qmd

```diff
@@ -211,7 +210,7 @@ returns_daily |>
 
 \index{Summary statistics}
 
-Note that you are now also equipped with all tools to download price data for *each* symbol listed in the S&P 500 index with the same number of lines of code. Just use `symbol <- tq_index("SP500")`, which provides you with a tibble that contains each symbol that is (currently) part of the S&P 500.\index{Data!SP 500} However, don't try this if you are not prepared to wait for a couple of minutes because this is quite some data to download!
+Note that you are now also equipped with all tools to download price data for *each* symbol listed in the S&P 500 index with the same number of lines of code. Just use `symbol <- download_data(type = "constituents", index = "S&P 500")`, which provides you with a tibble that contains each symbol that is (currently) part of the S&P 500.\index{Data!SP 500} However, don't try this if you are not prepared to wait for a couple of minutes because this is quite some data to download!
```

Comment:
Error in download_data(type = "constituents", index = "S&P 500") : 
  unused argument (index = "S&P 500")

Reply:
Must be because you don't have the most recent version installed because it 100% works with me on version 0.4.1